Dockerfile.tools

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/openshift/origin-release:golang-1.16 as build
+FROM registry.ci.openshift.org/openshift/release:golang-1.17 as build
```

Comment:
Please add a line to the commit message saying that we no longer mirror to docker.io/openshift/ and are using registry.ci.openshift.org/openshift/ as that is also public. Otherwise we will forget why we made this change.

Reply:
Sure, done.